cmd/lakectl/cmd/root.go

```diff
@@ -195,6 +203,32 @@ var rootCmd = &cobra.Command{
 	},
 }
 
+var statsNoErr = []string{
+	"lakectl_doctor",
+	"lakectl_config",
+}
+
+func sendStats(ctx context.Context, client api.ClientWithResponsesInterface, cmd string) {
+	resp, err := client.PostStatsEventsWithResponse(ctx, api.PostStatsEventsJSONRequestBody{
+		Events: []api.StatsEvent{
+			{
+				Class: "lakectl",
+				Name:  cmd,
+				Count: 1,
+			},
+		},
+	})
+
+	if slices.Contains(statsNoErr, cmd) {
+		return
+	}
+	if err != nil {
+		_, _ = fmt.Fprintf(os.Stderr, "Error sending statistics: %s\n", err)
+	} else if resp.StatusCode() != http.StatusNoContent {
+		_, _ = fmt.Fprintf(os.Stderr, "Failure sending statistics: Status Code: %d\n", resp.StatusCode())
+	}
+}
```

Comment:
Prefer to call it warning and not error and would simplify to just one message where we render the err or resp.Status() (both strings) to capture what went wrong while trying to post the data.